nabu-log-exporter/src/main/java/com/github/millerm/NabuLogExporter.java

```diff
@@ -0,0 +1,261 @@
+            this.stateFilePath = stateFilePath.toString() + "/" + filePath.getFileName().toString();
+            this.filePointer = readState();
+        }
+
+        private static String parseLog(String line) {
+            String[] parts = line.split(" ", 7);
+
+            // TODO: better parse
+            return "{" +
+                    "\"traceId\":\"" + parts[0] + "\"," +
+                    "\"spanId\":\"" + parts[1] + "\"," +
+                    "\"operationName\":\"" + parts[5] + "\"," +
+                    "\"references\":[]," +
+                    "\"startTimeUnixNano\":\"" + "1716525641" + "\"," +
+                    "\"endTimeUnixNano\":\"" + "1716525642" + "\"," +
+                    "\"kind\":2" +
+                    "}";
+
+        }
+
+        private static void handleLog(String line) {
+            HttpClient client = HttpClient.newHttpClient();
+            URI uri = URI.create("http://localhost:4318/v1/traces");
+
+            StringBuilder spansArray = new StringBuilder();
+            spansArray.append("[");
+            String span = parseLog(line);
+            spansArray.append(span);
+            spansArray.append("]");
+
+            // Wrap the spans array in a root JSON object
+            String requestBody = "{\"resourceSpans\":[" +
+                    "{\"resource\":{\"attributes\":[{\"key\":\"service.name\",\"value\":{\"stringValue\":\"nabu-tracer-tester2\"}}]},"
+                    +
+                    "\"scopeSpans\":[{\"scope\":{\"name\":\"manual-test\"},\"spans\":"
+                    + spansArray.toString() + "}]}]}";
+
+            LOG.info("Sending " + requestBody);
+
+            HttpRequest request = HttpRequest.newBuilder()
+                    .uri(uri)
+                    .header("Content-Type", "application/json")
+                    .POST(BodyPublishers.ofString(requestBody))
+                    .build();
+
+            try {
+                HttpResponse<String> response = client.send(request, BodyHandlers.ofString());
+                LOG.info("Response status code: " + response.statusCode());
+                LOG.info("Response body: " + response.body());
+            } catch (Exception e) {
+                e.printStackTrace();
+            }
+        }
+
+        @Override
+        public void run() {
+            try (RandomAccessFile reader = new RandomAccessFile(filePath.toFile(), "r")) {
+                String logEntry;
+
+                // Initially, set the reader to position read from state
+                reader.seek(filePointer);
+                while (true) {
+                    // NOTE(@millerm): Can switch to read() to read a specific amount
+                    // of bytes if necessary
+                    logEntry = reader.readLine();
+
+                    if (logEntry != null) {
+                        LOG.info("New log: " + logEntry);
+
+                        handleLog(logEntry);
+                        saveState(reader.getFilePointer());
```

Comment:
Is there a harm in pushing duplicate logs to the backend? May be we can saveState() in a background thread?